docs/thirdparty.rst

```diff
@@ -27,12 +28,15 @@ The following modules are availble to patch::
         'aiobotocore',
         'boto3',
         'botocore',
+        'pynamodb',
         'requests',
         'sqlite3',
         'mysql',
     )
 
-Patching boto3 and botocore are equivalent since boto3 depends on botocore
+Patching boto3 and botocore are equivalent since boto3 depends on botocore.
+
+Patching pynamodb patches botocore as well, as it depends partially on botocore.
```

Comment:
I would suggest put more details on what is patched for `pynamodb` so other users utilizing your work have more details and know the implication of patching `pynamodb`.

Reply:
I find it pretty difficult to write proper tests for the whole stuff, thanks to all the patching. I added a test now to check that only a single subsegment gets created when the pynamodb patch is enabled, but another AWS service is called. That single subsegment gets created, because the botocore patch is active in this case too.

Reply:
I extended the paragraph a bit. I hope that's sufficient.